src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonApiService.kt

```diff
@@ -73,4 +74,12 @@ class PrisonApiService(
       .retrieve()
       .awaitBody()
   }
+
+  suspend fun getActivePrisons(useClientCredentials: Boolean = false): List<Prison> {
```

Comment:
`useClientCredentials` is a bit of a mystery to me, but i know there's an effort to use "system" tokens and their roles. just leaving this note here to maybe explore using the `hmpps-incentives-api` token and giving it any extra roles if necessary

Reply:
This method has this boolean for consistency with the other methods in this class. But the `KpiService` will use the system token (by passing `true`)